jjb/static/scale-ci-pipeline.yml

```diff
@@ -13,6 +12,26 @@
         default: false
         description: ''
         name: PIPELINE
+    - bool:
+        default: true
+        description: 'runs router, object-density, cluster-density workloads typically at 25 node scale'
+        name: STAGE-TWO
```

Comment:
Should these names be "STAGE-TWO" or "STAGE_TWO" ? Same for the other stages

Reply:
nice catch!